### DISCUSSION TITLE
Server Time Out while stopping a notice several times

### DISCUSSION THREAD

**Discussion by lepo** [OP-TED/eForms-SDK#1053] [Q&A] [answered]
HI,

By stopping notices in the test environment we get a timeout from your gateway "504 Gateway Timeout ERROR" with the message 
"We can't connect to the server for this app or website at this time. There might be too much traffic or a configuration error. Try again later, or contact the app or website owner. If you provide content to customers through CloudFront, you can find steps to troubleshoot and help prevent this error by reviewing the CloudFront documentation." We tried it several times with the same result.

other notices can be stopped at the same time from the same system. 
Can you please explain where's the problem here.

Stopping Time btw tried to stop: 24.10.2024 between 17:00 - 18:00 an still not working
NoticeId is 9da83454-8fb9-471d-915b-49b4c5f0ee14

Please clearify and give us feedback. 
Thanks in advance,

Best
Günter

**Comment by YvesJo** [comment, marked as answer]
Hi,
For this non-SDK related question, please get in touch directly with the Helpdesk. Also please, in addition to the notice id, specify the version  of the notice you're trying to stop.
KR,
Yves
